lib/domain/core/i_settings_repository.dart

```diff
@@ -5,4 +5,10 @@ abstract class ISettingsRepository {
 
   // ignore: avoid_positional_boolean_parameters
   Future<void> setWasUserOnboarded({required bool wasOnboarded});
+
+  //* Get list of private crowdaction IDs the user has access to.
```

Comment:
Shouldn't this be in user repository?

Reply:
@Xazin are you referring to all the four functions or only to these two:
- `Future<void> setWasUserOnboarded({required bool wasOnboarded});`
- `Future<List<String>> getCrowdActionAccessList();`

Reply:
Also, I saw that most of the functions in the `UserRepository` are related to authentication so was confused as to what it was exactly meant for.